source/MapDetailPanel.cpp

```diff
@@ -672,6 +672,7 @@ void MapDetailPanel::DrawKey()
 // details, trade prices, and details about the selected object.
 void MapDetailPanel::DrawInfo()
 {
+	const Color &dimmer = *GameData::Colors().Get("dimmer");
 	const Color &dim = *GameData::Colors().Get("dim");
 	const Color &medium = *GameData::Colors().Get("medium");
```

Comment:
Looking elsewhere in the game, this does actually seem like a scenario where we'd use "faint" rather than "dimmer". The dimmer color is only used once, and in that one case it isn't be used on text.
And as Rising notes, there's not much of a difference between dim and dimmer.
![image](https://github.com/endless-sky/endless-sky/assets/17688683/30151a3b-e83e-4cc3-b689-dcae85c9f28a)
If the concern is that faint is too faint, we could slightly increase the brightness. Since the dimmer color isn't used for text anywhere, what we actually have is dim at 25% brightness and faint at 10%. We could bump faint up to something like 12% to make it easier to see; that screenshot is actually with faint changed to 12% instead of 10%.